modules/representation_management/spec/requests/v0/pdf_generator_2122a_spec.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'PdfGenerator2122aController', type: :request do
+  describe 'POST #create' do
+    let(:base_path) { '/representation_management/v0/pdf_generator2122a' }
+
+    context 'when submitting all required data' do
+      let(:all_required_params) do
```

Comment:
Can you define this in the `describe` block one level up so its available within both `context` blocks, then simply change the hash as necessary so you don't have to (mostly) duplicate it in the second `context` block?

Reply:
Done.